File: Linguagem C++/Estrutura Sequencial/Pagamento.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int horas;
    double valorHora,salario;
    string nome;

    cout << "Nome: ";
    getline(cin,nome);
    cout << "Valor por hora: ";
    cin >> valorHora;
    cout << "Horas trabalhadas: ";
    cin >> horas;

    salario = valorHora * horas;

    cout << fixed << setprecision(2);
    cout << "O pagamento para " << nome << " deve ser " << salario;


    return 0;
}

File: Linguagem C#/Sequencial/Pagamento.cs
using System;
using System.Globalization;

namespace Pagamento {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double valorHora, salario;
            int horas;
            string nome;

            Console.Write("Nome: ");
            nome = Console.ReadLine();
            Console.Write("Valor por hora: ");
            valorHora = double.Parse(Console.ReadLine(),CI);
            Console.Write("Horas trabalhadas: ");
            horas = int.Parse(Console.ReadLine());

            salario = horas * valorHora;

            Console.WriteLine("O pagamento para "+nome+" deve ser "+salario.ToString("F2", CI));
        }
    }
}
